webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -104,6 +112,32 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
     [workspaceId],
   );
 
+  const resourceQuotasList = useAsync(
+    async (canceller) => {
+      if (workspaceId === undefined) {
+        return NotLoaded;
+      }
+      try {
+        const resp = await getKubernetesResourceQuotas(
+          { id: workspaceId },
+          { signal: canceller.signal },
+        );
+        return resp.resourceQuotas;
+      } catch (e) {
+        if (!isNonK8RMError(e)) {
```

Comment:
clarification: do we want negation here?

Reply:
Yes. NonK8RMError is supposed to mean that this error was caused because the RM is something other than Kubernetes. If the error is anything besides this NonK8RMError we want to surface it